VRDR/DeathRecord.csproj

```diff
@@ -11,6 +11,7 @@
     <Authors>aholmes@mitre.org</Authors>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <LangVersion>latest</LangVersion>

```

Comment:
This does not appear to be necessary as it comes from the framework, please remove.